app/src/components/search/SearchComponent.vue

```diff
@@ -137,45 +139,65 @@ export default Vue.extend({
     ]),
     ...mapActions(["updateURIFromState"]),
 
+    /** Sets the values of the SearchComponent parts after a switch between national/international */
     changeInternational(): void {
       // clear radius and location on international change
       this.radius = this.locationSearchValue = "";
       // set default radius in store
       this.setSelectedRadius();
-      // unset location in stre
+      // unset location in store
       this.setSelectedLocation();
       // update search parameter in store
       this.setInternational(this.internationalValue);
       // execute serach if directsearch is enabled
-      if (this.direktsearch) {
+      if (this.directsearch) {
         this.executeSearch();
       }
     },
+    /**
+     * When pressing enter it sets the selected search value of a suggestion list as the value of the SearchBar.
+     * If directsearch is activated it will immediately start the search.
+     *
+     * @param {string} value: The selected value of the list.
+     */
     onSearchValueEnter(value: string): void {
       if (value != this.searchValue) this.searchValue = value;
-      if (this.direktsearch || value == this.oldValue) {
+      if (this.directsearch || value == this.oldValue) {
         this.executeSearch();
       }
       this.oldValue = value;
     },
+    /**
+     * When pressing enter it sets the selected location of the suggestion list as the value of the LocationSearchBar.
+     * If directsearch is activated it will immediately start the search.
+     *
+     * @param {string} value: The selected value of the list.
+     */
     onLocationValueEnter(value: string): void {
       if (value != this.locationSearchValue) this.locationSearchValue = value;
-      if (this.direktsearch || value == this.oldValue) {
+      if (this.directsearch || value == this.oldValue) {
         this.executeSearch();
       }
       this.oldValue = value;
     },
+    /**
+     * Sets the selected radius as the value of the RadiusSelect.
+     * If directsearch is activated it will immediately start the search.
+     *
+     * @param {string} value: The selected value of the list.
+     */
     onRadiusChanged(value: string): void {
       if (value != this.radius) {
         this.radius = value;
       }
       // update state search parameter in store
       this.setSelectedRadius(this.radius);
       // execute serach if directsearch is enabled
-      if (this.direktsearch) {
+      if (this.directsearch) {
         this.executeSearch();
       }
     },
+    /** Starts the search with the given parameters. They will be stored in store and in the URI. */
```

Comment:
They are not directy stored in the url, because the url is not a store. The url will be updated witch the changed parameter